src/main/scala/tools/partest/TestingInterface.scala

```diff
@@ -27,29 +27,8 @@ class Framework extends sbt.testing.Framework {
 /** Represents one run of a suite of tests.
  */
 case class Runner(args: Array[String], remoteArgs: Array[String], testClassLoader: ClassLoader) extends sbt.testing.Runner {
-  /** Returns an array of tasks that when executed will run tests and suites determined by the
```

Comment:
The doc are just copy/pasted from https://github.com/sbt/test-interface/blob/master/src/main/java/sbt/testing/Runner.java